crates/sui-framework/sources/math.move

```diff
@@ -64,4 +64,16 @@ module sui::math {
 
         (res as u64)
     }
+    
+    /// Return the value of a base raised to a power
+    public fun pow(base: u64, exponent: u64): u64 {    
```

Comment:
It's unlikely that we want the exponent to have the same width as the base, can you make it `u8`?
```suggestion
    public fun pow(base: u64, exponent: u8): u64 {    
```